README.md

```diff
@@ -659,7 +662,17 @@ By default, records are listed by `last_modified` descending order. You can set
 ```js
 client.bucket("blog").collection("posts")
   .listRecords({sort: "title"})
-  .then(result => ...);
+  .then(({data, next}) => {
+```
+
+#### Polling for changes
+
+To retrieve the list of records modified since a given timestamp, use the `since` option:
+
+```js
+client.bucket("blog").collection("posts")
+  .listRecords({since: (new Date().getTime() - 86400)}) // changes since yesterday
```

Comment:
You cannot use that. Timestamp are opaque, and should never be constructed by the client.

Do  instead `.listRecords({since: lastTimestamp)})`